specification/liftrqumulo/resource-manager/Qumulo.Storage/preview/2022-06-27-preview/Qumulo.Storage.json

```diff
@@ -0,0 +1,753 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Qumulo.Storage",
+    "version": "2022-06-27-preview",
+    "x-cadl-generated": [
+      {
+        "emitter": "@azure-tools/cadl-autorest"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "host": "management.azure.com",
+  "produces": [
+    "application/json"
```

Comment:
it would have been helpful if you had submitted the diffs between what was earlier reviewed in the old PR vs whats not reviewed as a separate commit to make it easy to review the differences. Other than the api version comment everything else looks fine.

Reply:
As our service is new and still in private preview, first time we are checking into this repo. so there is no old PR in this repo.